service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java

```diff
@@ -29,6 +34,13 @@
 
     private EncounterMapper encounterMapper;
     private CompositionMapper compositionMapper;
+    private ListMapper listMapper;
+
+    private static void addEntry(Bundle bundle, Resource resource) {
```

Comment:
please don't put private methods above public ones

Reply:
Code formatter moves it to here, but I can move it back